tests/libtransmission/copy-test.cc

```diff
@@ -57,7 +59,7 @@ class CopyTest : public SandboxedTest
         size_t buf_pos = 0;
         while (buf_pos < buf_len && bytes_remaining > 0)
         {
-            uint64_t const chunk_size = MIN(buf_len - buf_pos, bytes_remaining);
+            uint64_t const chunk_size = std::min(buf_len - buf_pos, bytes_remaining);
```

Comment:
You forgot to wrap `buf_len - buf_pos` in parentheses and type cast it to `uint64_t`; in other words, it should become `std::min((uint64_t)(buf_len - buf_pos), bytes_remaining);`

Reply:
Left argument both operands are `size_t` same as `uint64_t` on 64-bit. Hence i got no compiler warning on that.
Fixed now.